### PR TITLE
Update local-accounts.md

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #7146,  so  i replaced  this  sentence **User Account Control: Run all administrators in Admin Approval Mode**    with 

this new sentence  **User Account Control: Admin Approval Mode for the Built-in Administrator account**

so this PR must be checked twice or thrice before merging. Also, the style format should be checked.

### CODE REVIEW

File: windows/security/identity-protection/access-control/local-accounts.md [hunk 1]
```diff
@@ -531,7 +532,7 @@ The following table shows the Group Policy settings that are used to deny networ
 
 11. Create links to all other OUs that contain servers.
 
-    **Note**  
+    **Note** 
     You might have to create a separate GPO if the user name of the default Administrator account is different on workstations and servers.
```
Comment (illfated) [inline windows/security/identity-protection/access-control/local-accounts.md:536]: May as well convert this note to a colored Note blob:
```suggestion
    > [!NOTE]
    > You might have to create a separate GPO, if the user name of the default Administrator account is different on workstations and servers.
```

File: windows/security/identity-protection/access-control/local-accounts.md [hunk 2]
```diff
@@ -309,7 +309,8 @@ The following table shows the Group Policy and registry settings that are used t
 <tr class="odd">
 <td><p>1</p></td>
 <td><p>Policy name</p></td>
-<td><p><a href="/windows/device-security/security-policy-settings/user-account-control-run-all-administrators-in-admin-approval-mode" data-raw-source="[User Account Control: Run all administrators in Admin Approval Mode](/windows/device-security/security-policy-settings/user-account-control-run-all-administrators-in-admin-approval-mode)">User Account Control: Run all administrators in Admin Approval Mode</a></p></td>
+<td><p><a href="/windows/security/threat-protection/security-policy-settings/user-account-control-admin-approval-mode-for-the-built-in-administrator-account" data-raw-source="[
+User Account Control: Admin Approval Mode for the Built-in Administrator account](/windows/security/threat-protection/security-policy-settings/user-account-control-admin-approval-mode-for-the-built-in-administrator-account)">User Account Control: Admin Approval Mode for the Built-in Administrator account</a></p></td>
```
Comment (illfated) [inline windows/security/identity-protection/access-control/local-accounts.md:313]: I don't think we need to add a line break (NewLine) here, so I suggest removing the line break:
```suggestion
<td><p><a href="/windows/security/threat-protection/security-policy-settings/user-account-control-admin-approval-mode-for-the-built-in-administrator-account" data-raw-source="[User Account Control: Admin Approval Mode for the Built-in Administrator account](/windows/security/threat-protection/security-policy-settings/user-account-control-admin-approval-mode-for-the-built-in-administrator-account)">User Account Control: Admin Approval Mode for the Built-in Administrator account</a></p></td>
```

**Comment by illfated** [comment]
#sign-off

**Comment by RAJU2529** [comment]
@e0i .  Please look into this PR #7147

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/access-control/local-accounts.md [hunk 3]
```diff
@@ -531,8 +531,8 @@ The following table shows the Group Policy settings that are used to deny networ
 
 11. Create links to all other OUs that contain servers.
 
-    **Note**  
-    You might have to create a separate GPO if the user name of the default Administrator account is different on workstations and servers.
+    > [!NOTE]
+    > You might have to create a separate GPO, if the user name of the default Administrator account is different on workstations and servers.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/local-accounts.md:535]:
```suggestion
    > You might have to create a separate GPO if the user name of the default Administrator account is different on workstations and servers.
```

Comment (JohanFreelancer9) [reply windows/security/identity-protection/access-control/local-accounts.md:535]: The comma is not needed here - please remove.

**Comment by e0i** [comment]
Replaced link targets are broken. Closing.

**Pull request closed without merging**